README.md

```diff
@@ -165,28 +165,27 @@ Then register your custom adapter:
 
 ```java
 CustomTypeAdapter<Date> customTypeAdapter = new CustomTypeAdapter<Date>() {
-    @Override
-    public Date decode(String value) {
-        try {
-            return ISO8601_DATE_FORMAT.parse(value);
-        } catch (ParseException e) {
-            throw new RuntimeException(e);
-        }
+  @Override
+  public Date decode(String value) {
+    try {
+      return ISO8601_DATE_FORMAT.parse(value);
+    } catch (ParseException e) {
+      throw new RuntimeException(e);
     }
+  }
 
-    @Override
-    public String encode(Date value) {
-        return ISO8601_DATE_FORMAT.format(value);
-    }
+  @Override
+  public String encode(Date value) {
+    return ISO8601_DATE_FORMAT.format(value);
+  }
 };
 
 // use on creating ApolloClient
 ApolloClient.builder()
-    .serverUrl(serverUrl)
-    .okHttpClient(okHttpClient)
-    .normalizedCache(normalizedCacheFactory, cacheKeyResolver)
```

Comment:
Intentional removal of cache setup?

Reply:
Yeah, as it's snippet just for custom type adapters